pkg/controller/artifacts/crd-network.yaml

```diff
@@ -31,6 +31,17 @@ spec:
               type: string
             vpcID:
               type: string
+            service:
+              type: object
+              properties:
+                ipam:
+                  type: string
+                  enum: ["Kubernetes", "External"]
```

Comment:
Minor: "kubernetes", or "Arktos"? I prefer "Arktos" as ae are forking away from K8s.

Reply:
what matters is non-External and External; however, adding Arktos also makes sense. Will add "Arktos" too.